src/gdcdictionary/schemas/pathology_detail.yaml

```diff
@@ -1012,6 +1057,60 @@ properties:
     maximum: 100
     minimum: 0
 
+  spindle_cell_percent_range:
+    $ref:
+      - "_terms.yaml#/spindle_cell_percent/common"
+    enum:
+      - 0%
+      - 1-30%
+      - 31-60%
+      - 61-90%
+      - '>90%'
+      - Unknown
+      - Not Reported
+
+  timepoint_category:
+    $ref:
+      - "_terms.yaml#/timepoint_category/common"
+    enum:
+      - Adjuvant Therapy
+      - Adolescence
+      - Adulthood
+      - After Chemotherapy
+      - After Study Enrollment
+      - Childhood
+      - End of Consolidation Therapy
+      - End of Treatment Course
+      - End of Treatment Course 1
+      - End of Treatment Course 2
+      - First Complete Response
+      - First Treatment
+      - Follow-up
```

Comment:
Should this be "Follow-Up"?  We can keep it lowercase if the other nodes have it like this.

Reply:
Yeah the letter case was set on the FollowUp node funny enough, which was then inherited by OtherClinicalAttribute. I think for sake of consistency we should keep as is and for other nodes, and then we can deprecate+migrate in the future.